code/modules/projectiles/guns/ego_gun/aleph.dm

```diff
@@ -3,12 +3,14 @@
 	desc = "The star shines brighter as our despair gathers. The weapon's small, evocative sphere fires a warm ray."
 	icon_state = "star"
 	inhand_icon_state = "star"
+	special = "This gun fires 240 rounds per minute. \
+			This gun scales with remaining SP."
 	ammo_type = /obj/item/ammo_casing/caseless/ego_star
 	weapon_weight = WEAPON_HEAVY
 	spread = 5
 	fire_sound = 'sound/weapons/ego/star.ogg'
 	vary_fire_sound = TRUE
-	fire_sound_volume = 50
+	fire_sound_volume = 25
```

Comment:
But why :thinking:

Reply:
It hurt my ears with how loud it was, while everything else was normal